typings/react-datetime-tests.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
```

Comment:
I was following the [recommendations](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package) from the DefinitelyTyped repository. This one doesn't necessarily apply here, I can move it.